app/components/forms/orders/order-form.js

```diff
@@ -497,7 +497,10 @@ export default Component.extend(FormMixin, {
   },
 
   allFields: computed('fields', function() {
-    return groupBy(this.fields.toArray(), field => field.get('form'));
+    const requiredFixed = this.fields.toArray().slice(0, 3);
```

Comment:
There is no guarantee that only first three fields are fixed fields, the logic is flawed

Reply:
this is for firstName, lastName and email. They are always at top?

Reply:
the rest in order in which they are in attendee step

Reply:
But they'll not always be 3